### ISSUE TITLE
The network I have trained for one day, but all parameter in m2pelu layer  still keep value  ‘1’.  Who can help me explain this phenomenon？

### ISSUE THREAD

**Issue by xiaoxiaomuyu0** [Coldmooon/Code-for-MPELU#2]
layer {
  name: "m2pelu_conv1"
  type: "M2PELU"
  bottom: "conv1"
  top: "conv1"
  param {
    lr_mult: 10
    decay_mult: 0
  }
  param {
    lr_mult: 10
    decay_mult: 0
  }
  m2pelu_param {
    alpha_filler {
      type: "constant"
      value: 1
    }
    beta_filler {
      type: "constant"
      value: 1
    }
  }
}

base_lr: 0.0001
lr_policy: "multistep"
gamma: 0.1
stepvalue: 100000
stepvalue: 600000
momentum: 0.9
weight_decay: 0.0005
max_iter: 930000

**Comment by xiaoxiaomuyu0** [comment]
Sorry, three-layer network can work normally and have slight changes in m2pelu-parameters, maybe I should adjust parameters carefully in twenty-layer.

**Comment by cookyecat** [comment]
@xiaoxiaomuyu0 The base learning rate of 0.0001 is too small, which will result in very small gradients. Please try larger learning rate.
